apps/buildog/src/app/organizations/page.tsx

```diff
@@ -0,0 +1,137 @@
+"use client";
+
+import React, { useEffect, useState } from "react";
+import { Button } from "@repo/ui/components/ui/button";
+import {
+  Dialog,
+  DialogContent,
+  DialogDescription,
+  DialogFooter,
+  DialogHeader,
+  DialogTitle,
+  DialogTrigger,
+} from "@repo/ui/components/ui/dialog";
+import { Input } from "@repo/ui/components/ui/input";
+import { Label } from "@repo/ui/components/ui/label";
+import { useRouter } from "next/navigation";
+import getOrganizations from "@/lib/get-organizations";
+import { Card } from "@repo/ui/components/ui/card";
+import { ArrowRightIcon, PlusCircledIcon } from "@ui/components/ui/react-icons";
+
+export default function Page() {
+  const [organizations, setOrganizations] = useState([]);
+  const [newOrgName, setNewOrgName] = useState("");
+  const router = useRouter();
+
+  useEffect(() => {
+    async function fetchOrganizations() {
+      const orgs = await getOrganizations();
+      setOrganizations(orgs);
+    }
+
+    fetchOrganizations();
+  }, []);
+
+  const handleCreateOrganization = async () => {
+    console.log("Creating organization:", newOrgName);
+
+    // We will add creating tenants here when the backend is ready - Furkan
```

Comment:
You don't need to take some notes with comment. If you want to take notes for future task you can open an issue and remark the files. For example, "connect handleCreateOrganization to backend, [page link]"